frontend/packages/pipelines-plugin/src/components/shared/results/ResultsList.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Table, TableHeader, TableBody } from '@patternfly/react-table';
+import { SectionHeading } from '@console/internal/components/utils';
+import { TektonResultsRun } from '../../../types';
+import { getCellsFromResults } from '../../../utils/pipeline-utils';
+
+export interface ResultsListProps {
+  results: TektonResultsRun[];
+  resourceName: string;
+}
+
+const ResultsList: React.FC<ResultsListProps> = ({ results, resourceName }) => {
+  const { t } = useTranslation();
+  if (!results.length) return null;
+
+  const { rows, columns } = getCellsFromResults(results);
+
+  return (
+    <>
+      <SectionHeading text={t('pipelines-plugin~{{resourceName}} Results', { resourceName })} />
+      <Table
```

Comment:
Did you push code to make the alert appear? I don't see it 😕

Reply:
My bad.. updated now